### ISSUE TITLE
console.log in renderer process will output different backslash in spectron in 1.6.x compare to 1.4.x

### ISSUE THREAD

**Issue by jwu** [electron/electron#8886]
* Electron version: 1.6.x
* Operating system: Mac and Windows

### Expected behavior

The output logs' backslash count should be same as 1.4.x

### Actual behavior

More backslashes added to the ouputs

### How to reproduce

```bash
git clone https://github.com/jwu/electron-backslash-issue
npm install
npm start
```

This bug relate with spectron, I can not distinguish if this is a bug from spectron or electron. Since I use the same versions of spectron with different version of electron to get the different output, I think it should be electron's issue first.

In short, the output come from spectron's api `app.getRenderProcessLogs()` will be different when using electron 1.6.x (compare to electron 1.4.x). In 1.6.x the output will add additional backslash which will make the test in spectron i used failed.

I've create a repo to reproduce the issue: https://github.com/jwu/electron-backslash-issue.

**Comment by MarshallOfSound** [comment]
A quick experimentation makes me believe this was an intended change as a result of the Chromium update.  Things like this will change over time, will defer to someone to confirm this suspicion though 👍

**Comment by kevinsawicki** [comment]
Yeah, the format Chrome uses can and will change over time. The spectron specs themselves initialized failed on 1.6 because of these changes, https://github.com/electron/spectron/pull/162/files#diff-e42eccb97686f30281e1d78eeedd5d06

You could raise this issue to the Chrome issue tracker if you want to discuss the root cause of these format changes and what stability you can expect going forward.

I'm going to close this out here though since this log format is not something Electron or spectron has control over.
